src/main/src/main/java/org/vfny/geoserver/util/DataStoreUtils.java

```diff
@@ -320,23 +320,27 @@ else if (!resourceInfo.getStore().getType().equalsIgnoreCase("Web Feature Server
 
     // A utility method for retreiving supported SRS on WMTSLayerInfo resource
     public static List<String> getOtherSRSFromWMTSStore(WMTSLayerInfo wmtsLayerInfo) {
+        List<String> otherSRS = new ArrayList<>();
         try {
-
             Layer wmtsLayer = wmtsLayerInfo.getWMTSLayer(new NullProgressListener());
-
             Set<String> supportedSRS = wmtsLayer.getSrs();
-            // check if there are additional srs available
-            // if not return an empty list for legacy behavior
-            if (supportedSRS.size() == 1) return Collections.EMPTY_LIST;
-            List<String> otherSRS = supportedSRS.stream().collect(Collectors.toList());
-            return otherSRS;
+            String urnFormat = "urn:ogc:def:crs:EPSG::";
```

Comment:
What happens when these fixed SRS names are used for cascading? Wondering if it might lead to an axis flip issue.

Reply:
thanks for your point @aaime . Fixed this by leaving the urn format for CRS.